src/app/story/storyrow/storyrow.component.html

```diff
@@ -65,3 +65,10 @@
     </div>
   </div>
 </div>
+<div class="node-siblings-column story-row-column total-time-column" *ngIf="first">
+  <app-totaltimes [ngClass]="'total-times'"
```

Comment:
Could you use `class="total-times"`? I don't think `ngClass` is necessary since it's a static value.